commit dc8bd7ed080d3ad716b5d9dc065c3262a258e141
Author: Randy Dunlap <rdunlap@infradead.org>
Date:   Fri Dec 28 00:31:39 2018 -0800

    arch/sh/boards/mach-kfr2r09/setup.c: fix struct mtd_oob_ops build warning
    
    [ Upstream commit 440e7b379f91acd245d5c8de94d533f40f5dffb3 ]
    
    arch/sh/boards/mach-kfr2r09/setup.c does not need to #include
    <mtd/onenand.h>, and doing so causes a build warning, so drop that header
    file.
    
    In file included from ../arch/sh/boards/mach-kfr2r09/setup.c:28:
    ../include/linux/mtd/onenand.h:225:12: warning: 'struct mtd_oob_ops' declared inside parameter list will not be visible outside of this definition or declaration
         struct mtd_oob_ops *ops);
    
    Link: http://lkml.kernel.org/r/702f0a25-c63e-6912-4640-6ab0f00afbc7@infradead.org
    Fixes: f3590dc32974 ("media: arch: sh: kfr2r09: Use new renesas-ceu camera driver")
    
    Signed-off-by: Randy Dunlap <rdunlap@infradead.org>
    Reported-by: Geert Uytterhoeven <geert@linux-m68k.org>
    Suggested-by: Miquel Raynal <miquel.raynal@bootlin.com>
    Reviewed-by: Miquel Raynal <miquel.raynal@bootlin.com>
    Cc: Yoshinori Sato <ysato@users.sourceforge.jp>
    Cc: Rich Felker <dalias@libc.org>
    Cc: Jacopo Mondi <jacopo+renesas@jmondi.org>
    Cc: Magnus Damm <magnus.damm@gmail.com>
    Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
    Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/arch/sh/boards/mach-kfr2r09/setup.c b/arch/sh/boards/mach-kfr2r09/setup.c
index e59c577ed871..c70bc7809dda 100644
--- a/arch/sh/boards/mach-kfr2r09/setup.c
+++ b/arch/sh/boards/mach-kfr2r09/setup.c
@@ -25,7 +25,6 @@
 #include <linux/memblock.h>
 #include <linux/mfd/tmio.h>
 #include <linux/mmc/host.h>
-#include <linux/mtd/onenand.h>
 #include <linux/mtd/physmap.h>
 #include <linux/platform_data/lv5207lp.h>
 #include <linux/platform_device.h>
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1217_dc8bd7ed080d3ad716b5d9dc065c3262a258e141.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1217_dc8bd7ed080d3ad716b5d9dc065c3262a258e141.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
